src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTest.razor

```diff
@@ -2,12 +2,12 @@
     <BitButton OnClick="IncrementCount"
                IsEnabled="IsEnabled"
                ButtonStyle="ButtonStyle"
-               AriaDescription="AriaDescription"
-               AriaLabel="AriaLabel"
-               AriaHidden="AriaHidden"
-               Href="Href"
-               Title="Title"
-               Target="Target">
+               AriaDescription="@AriaDescription"
+               AriaLabel="@AriaLabel"
+               AriaHidden="@AriaHidden"
```

Comment:
the "AriaHidden" parameter does not require the "@" in its value, because its type is not string